ui/raidboss/data/07-dt/raid/r5s.ts

```diff
@@ -616,70 +616,142 @@ const triggerSet: TriggerSet<Data> = {
   ],
   timelineReplace: [
     {
-      locale: 'en',
-      replaceText: {
-        'Flip to A-side/Flip to B-side': 'Flip to A/B-side',
-        'Play A-side/Play B-side': 'Play A/B-side',
+      'locale': 'en',
+      'replaceText': {
         '2-snap Twist & Drop the Needle/3-snap Twist & Drop the Needle/4-snap Twist & Drop the Needle':
           '2/3/4-snap Twist',
+        'Flip to A-side/Flip to B-side': 'Flip to A/B-side',
+        'Play A-side/Play B-side': 'Play A/B-side',
       },
     },
     {
-      'missingTranslations': true,
       'locale': 'de',
+      'missingTranslations': true,
       'replaceSync': {
         'Dancing Green': 'Springhis Khan',
         'Frogtourage': 'Schenkelschwinger',
       },
-      'replaceText': {},
+      'replaceText': {
+        '\\(Cleave\\)': '(Cleave)',
+        '\\(Echo\\)': '(Echo)',
+        '\\(In+Protean+Echo\\)': '(Rein+Himmelsrichtungen+Echo)',
+        '\\(Out+Protean+Echo\\)': '(Raus+Himmelsrichtungen+Echo)',
+        '\\(Out+Protean\\)': '(Raus+Himmelsrichtungen)',
```

Comment:
`+` is also regex 🤔

Reply:
yeah its not added in my script for automatic escaping, thanks for spotting